source/components/tooltip/tooltip-docs.less

```diff
@@ -1 +1,32 @@
-// TBD
+@import 'vars';
+
+.tooltip-examples {
+  hx-tooltip {
+    display: inline-block;
+    margin: 1em;
+    position: relative;
+  }
+}
+
+.demo-target {
+  border: 3px dotted #000000;
+  margin: auto;
+  min-width: 200px;
+  padding: 10px;
+  text-align: center;
+  width: 50%;
+  hx-icon {
+    color: @gray-700;
+    vertical-align: text-top;
+  }
+}
+
+.demo {
+  align-items: center;
+  display: flex;
+  justify-content: center;
+  min-height: 6rem;
+  & > p {
```

Comment:
our `p` element is exaclty vertically aligned because of `p` is styled to have a 10px margin-bottom.

I'd either add `margin: 0;` here or not have the hover demo inside a `p` to get it vertically aligned properly.